src/components/layout/Section.tsx

```
@@ -0,0 +1,112 @@
+    },
+    ref
+  ) => {
+    const getSafeAreaClass = () => {
+      if (!safeArea) return ''
+      if (typeof safeArea === 'boolean') return safeAreaClasses.all
+      return safeAreaClasses[safeArea]
+    }
+
+    return (
+      <Component
+        ref={ref as any}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)